masterlist.yaml

```
@@ -7778,6 +7778,16 @@ plugins:
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         nav: 10
 
+  - name: 'Complementary Grass Fixes.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/71562'
```

Links should have trailing slashes if possible:

```suggestion
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/71562/'
```